rpc/api.proto

```diff
@@ -724,10 +726,11 @@ message TicketBuyerConfigResponse{
     double max_price_relative = 15;
     int64 max_in_mempool = 17;
     string pool_address = 18;
-    double pool_fees = 19;
-    bool spread_ticket_purchases = 20;
-    string voting_address = 21;
+	double pool_fees = 19;
```

Comment:
Indentation issue still persists. Maybe this will help, this is what it looks like viewing the file directly: https://github.com/shweini/dcrwallet/blob/66b13d5c74e3daaa42253ed19669c8e259328497/rpc/api.proto#L729